pkg/volume/flexvolume/driver-call.go

```diff
@@ -219,6 +223,7 @@ type DriverStatus struct {
 	Capabilities *DriverCapabilities `json:",omitempty"`
 }
 
+// DriverCapabilities represents what driver an do
```

Comment:
*driver can do

Reply:
Fixed. @chakri-nelluri  could you give a ok-to-test, so that bot can start testing? thx